libraries/AdaptiveExpressions/BuiltinFunctions/ConvertFromUtc.cs

```diff
@@ -23,24 +25,30 @@ private static (object value, string error) Evaluator(Expression expression, IMe
             object value = null;
             string error = null;
             IReadOnlyList<object> args;
+            var locale = options.Locale != null ? new CultureInfo(options.Locale) : Thread.CurrentThread.CurrentCulture;
```

Comment:
nit, in general I would call the variable and parameter cultureInfo rather than locale so it doesn't get confused with the locale string we use in the other places.

Reply:
Agree. One reason I choose locale is I want to use same property name in both C# and JS. 
Will discuss with Dong to choose a more accurate pamameter name here.